apps/src/applab/designMode.js

```diff
@@ -641,7 +642,9 @@ designMode.changeThemeForCurrentScreen = function(prevThemeValue, themeValue) {
       const currentPropValue = designMode.readProperty(element, propName);
       const {type} = propTheme;
       let propShouldUpdate;
-      if (type === 'color') {
+      if (currentPropValue === '') {
+        propShouldUpdate = true;
```

Comment:
This might be worth a comment. Do we have an explicit expectation that no theme value should ever be unset, unless it's a pre-themes project?

Reply:
agree, I will add a comment. That was my intention - we are backfilling all empty values so that we no longer rely on upstream CSS for themeable properties. they should always be present in the inline styles.